glimpseClientVersion.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
```

Comment:
I thought about that, but thought that since it changes rarely, having it checked in would be provide more visibility to the file and when it changes.